shell/platform/windows/client_wrapper/include/flutter/flutter_view.h

```diff
@@ -0,0 +1,46 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_WINDOWS_CLIENT_WRAPPER_INCLUDE_FLUTTER_FLUTTER_VIEW_H_
+#define FLUTTER_SHELL_PLATFORM_WINDOWS_CLIENT_WRAPPER_INCLUDE_FLUTTER_FLUTTER_VIEW_H_
+
+#include <flutter_windows.h>
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "plugin_registrar.h"
+
+namespace flutter {
+
+class FlutterViewWin32;
+
+typedef std::shared_ptr<FlutterViewWin32> FlutterView;
+
+// A view for displaying Flutter content.
```

Comment:
Could you flesh this comment out? The window API was fairly self-contained, so it was more obvious what it was. It's less obvious what exactly this class is and how to use it, so more explanation of how it's used would be helpful.

Reply:
This object has gone away post combining view and view controller.